runtime/crt-util/jvm/test/HttpTest.kt

```diff
@@ -3,8 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package aws.smithy.kotlin.runtime.crt
-
+import aws.smithy.kotlin.runtime.crt.update
```

Comment:
It seems like a bug in my IDE, it gives this error: `Package directive does not match the file location` and then the suggested action removes the `package` entirely. lol.

Reply:
That's probably because the file location is **runtime/crt-util/jvm/test**. It should be moved further down to **runtime/crt-util/jvm/test/aws/smithy/kotlin/runtime/crt**.